presto-docs/src/main/sphinx/functions/qdigest.rst

```diff
@@ -51,3 +51,9 @@ Functions
     the per-item weight ``w`` and maximum error of ``accuracy``. ``accuracy``
     must be a value greater than zero and less than one, and it must be constant
     for all input rows.
+
+.. function:: truncated_mean(qdigest(T), lower_quantile, upper_quantile) -> double
```

Comment:
Shouldn't the return type be `T`?

Reply:
This was a bit of a dilemma, since the mean of a set of longs is a double, as is the mean of doubles, but the mean of reals is a real. So the return type is technically a union of double and real. What's the best way of conveying that here?

Reply:
@blrnw3 Perhaps, put `-> double or real` in the signature and explain more in the docs.